CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+Contributing to WildFly Elytron
+==================================
+- [Forking the Project](#forking-the-project)
+- [Issues](#issues)
+    * [Good First Issues](#good-first-issues)
+- [Setting up your Developer Environment](#setting-up-your-developer-environment)
+- [Coding Guidlines](#coding-guidlines)
+
+
+## Forking the Project 
+To contribute, you will first need to fork the [wildfly-elytron](https://github.com/wildfly-security/wildfly-elytron) repository. 
+
+This can be done by looking in the top-right corner of the repository page and click "Fork".
+![fork](assets/images/fork.jpg)
+
+The next step is to clone your newly forked repository onto your local workspace. This can be done by going to your newly forked repository, which should be at `https://github.com/USERNAME/wildfly-elytron`. 
+
+Then, there will be a green button that says "Code". Click on that and copy the URL.
+
+![clone](assets/images/clone.png)
+
+Then, in your terminal, paste the following command:
+`git clone [URL]`
+Be sure to replace [URL] with the URL that you copied.
+
+Now you have the repository on your computer!
+
+## Issues
+The WildFly Elytron project uses JIRA to manage issues. All issues can be found [here](https://issues.redhat.com/projects/ELY/issues). 
+
+To create a new issue, comment on an existing issue, or assign an issue to yourself, you'll need to first [create a JIRA account](https://issues.redhat.com/).
+
+
+### Good First Issues
+Want to contribute to the WildFly Elytron project but aren't quite sure where to start? Check out our issues with the `good-first-issue` label. These are a triaged set of issues that are great for getting started on our project. These can be found [here](https://issues.redhat.com/issues/?filter=12383825). 
+
+Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. Then, remember to assign it to yourself, by clicking on "Assign to me", to prevent someone else from also working on the same issue.
+
+![jira](assets/images/jira.png)
+
+It is recommend that you use a separate branch for every Pull Request to submit. To keep things straightforward and memorable, you should try to name the branch the JIRA issue number. Doing so also allows your to prevent conflicts in the future when working on other issues. For example, for ELY-2127, you should also name the branch ELY-2127.
```

Comment:
s/recommend/recommended

s/Pull Request to submit/issue you work on

s/you should try to name the branch the JIRA issue number/you can name each branch using the JIRA issue number

s/Doing so also allows your to prevent conflicts in the future when working on other issues/This way, you can have multiple PRs open for different issues.

s/For example, for ELY-2127, you should also name the branch ELY-2127/For example, if you were working on [ELY-2127](https://issues.redhat.com/browse/ELY-2127), you could use ELY-2127 as your branch name.

Reply:
Changed